### PR TITLE
update organization description to match image

### PR DESCRIPTION
**Pull request by BlakeTurner**

### CODE REVIEW

**Comment by vweller** [comment]
On line 36 you repeated a word in "primarily **through** through the reduction of technical debt"

File: how-were-organized.md [hunk 1]
```diff
@@ -23,12 +25,16 @@ Finally, we have management groups who oversee the entire organization. They are
...
 
 ### Platform
 
 The Platform team owns our cloud infrastructure, our build pipeline, and our development environment. They play a big role in enabling Product Engineering, IT, and Data with infrastructural support and guidance. 
 
+#### Core
+
+The Core team maintains and improves our in-house software and its infrastructure. Their team's primary concern is Keeping the Lights On (KTLO) at the application level, their secondary concern is supporting stakeholder requests that are "too small for a pitch", and their third concern is enabling the speed and quality of our engineering, primarily through through the reduction of technical debt. The Core team works closely with BAS (Bench App Support) and BizTech to handle the issues, outages, and stakeholder requests that come our way during a cycle and its cooldown. In effect, the Core team makes the Product Engineering cycles _possible_ by absorbing the myriad distractions the team needs to deal with during a cycle. The number of engineers on the Core team scales based on total number of clients, a proxy for business complexity.
```
Comment (vweller) [inline how-were-organized.md:36]: Would like to change "their" to the in the second sentence "**Their** team's primary concern is"

Comment (jordanhudson) [inline how-were-organized.md:36]: If you say primary and secondary, it follows to say tertiary rather than third.

Comment (jordanhudson) [inline how-were-organized.md:36]: I don't see the word "Innovation"?  Or how it's mainly to reduce the effort spent to successfully KTLO.

Comment (BlakeTurner) [reply how-were-organized.md:36]: Hah i KNEW someone would comment on that

**Review by jordanhudson — APPROVED** [review]

**Pull request merged**